test/unit/player.test.js

```diff
@@ -835,6 +835,39 @@ test('createModal()', function() {
   strictEqual(player.children().indexOf(modal), -1, 'modal was removed from player\'s children');
 });
 
+test('should return correct remote text track values', function () {
```

Comment:
I think it probably would be better to move this test into `test/tracks/tracks.test.js`